201901_gaussian_fg_lensed_cmb_realistic_noise/simulation.cfg

```diff
@@ -0,0 +1,77 @@
+
+output_nside = 4096
+output_folder = output/$output_nside
+channels = all
```

Comment:
## Bandpass

For now only delta function at the reference frequency. 
If this is not good enough it would be easy to use PySM to run top-hat.